### PR TITLE
Done ex5_js-functions

### PR DESCRIPTION
**Pull request by SkyNicarum**

### CODE REVIEW

**Comment by SkyNicarum** [comment]
I have a problem with the fact that from the first time you can’t get filled because Travis fails and does not tick, and from the second everything is fine

**Comment by aneelia** [comment]
>  you can’t get filled 

> Travis fails and does not tick

what do u mean?

File: src/ex5_js-functions/task.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const Calculator = {
+    result: 0,
+    add: function addF(value = 0) {
```
Comment (aneelia) [inline src/ex5_js-functions/task.js]: > function addF

why did u use the functional named expression? u don't need it

File: src/ex5_js-functions/task.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    Calculator.result += value;
+    return addF;
```
Comment (aneelia) [inline src/ex5_js-functions/task.js]: u need to return `add`. use a recursion

File: src/ex5_js-functions/task.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    subtract: function subtractF(value = 0) {
```
Comment (aneelia) [inline src/ex5_js-functions/task.js]: make it better please, like this:
`divide: function divideF(value = 1) {
    Calculator.result /= value;
    return divideF;
},`

File: src/ex5_js-functions/task.js (hunk 1 above)
Comment (SkyNicarum) [reply src/ex5_js-functions/task.js]: Сan I just write a function or didn’t I understand something?

File: src/ex5_js-functions/task.js (hunk 3 above)
Comment (SkyNicarum) [reply src/ex5_js-functions/task.js]: What do u mean?

**Pull request closed without merging**